### DISCUSSION TITLE
Scheduled workflows: custom branch workaround?

### DISCUSSION THREAD

**Discussion by inwardmovement** [community/community#25101] [Actions]
<p>The <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule" rel="nofollow noopener">doc</a> says:</p>
<p><em>Scheduled workflows run on the latest commit on the default or base branch</em></p>
<p>Will it be made possible someday to perform a Scheduled trigger on a non-default branch?</p>
<p>This would allow more flexibilty and defining a “default branch” for other reasons than this limitation.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>I’ve found the same feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>In addition, you could found github action changes in github changelog: <a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.blog%2Fchangelog%2F&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C2522a736b2b343b0faab08d789e48f84%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637129487060276318&amp;sdata=%2FJE0hDFoQU0pjzr3yc%2Fl35saokDm%2Fm4uOZEf2J1KggQ%3D&amp;reserved=0" rel="nofollow noopener">https://github.blog/changelog/</a>.</p>

**Comment by inwardmovement** [comment]
<p>Thanks!</p>
<p>Not really related but is the internal feature request list / roadmap available to the public? This may prevent asking questions here that already have answers, thus saving time for your team in the first place.</p>
